Control_Structures_and_Function/Exceptions/useful_exceptions.py

```diff
@@ -0,0 +1,42 @@
+"""
+- Avoid “swallowing” useful exceptions with bare except clauses.
+- Programmers often make the mistake of overly relying on generic exception handling, especially when using third-party packages.
+- This leads to a loss of crucial debugging information.
+"""
+
+# You shouldn't do like this
+import requests
```

Comment:
The import statement should be at the top of the file and after the docstring.

Reply:
Done